tools/clang/test/SemaHLSL/hlsl/workgraph/ast-EmptyNodeOutputArrayTypes.hlsl

```diff
@@ -0,0 +1,74 @@
+// CHECK-NEXT:| |-FunctionTemplateDecl 0x{{.+}} <<invalid sloc>> <invalid sloc> GroupIncrementOutputCount
+// CHECK-NEXT:| | |-TemplateTypeParmDecl 0x{{.+}} <<invalid sloc>> <invalid sloc> class TResult
+// CHECK-NEXT:| | |-TemplateTypeParmDecl 0x{{.+}} <<invalid sloc>> <invalid sloc> class Tcount
+// CHECK-NEXT:| | |-CXXMethodDecl 0x{{.+}} <<invalid sloc>> <invalid sloc> implicit GroupIncrementOutputCount 'TResult (Tcount) const'
+// CHECK-NEXT:| | | `-ParmVarDecl 0x{{.+}} <<invalid sloc>> <invalid sloc> count 'Tcount'
+// CHECK-NEXT:| | `-CXXMethodDecl 0x[[GroupIncrementOutputCount:[0-9a-f]+]] <<invalid sloc>> <invalid sloc> used GroupIncrementOutputCount 'void (unsigned int)' extern
+// CHECK-NEXT:| |   |-TemplateArgument type 'void'
+// CHECK-NEXT:| |   |-TemplateArgument type 'unsigned int'
+// CHECK-NEXT:| |   |-ParmVarDecl 0x{{.+}} <<invalid sloc>> <invalid sloc> GroupIncrementOutputCount 'unsigned int'
+// CHECK-NEXT:| |   `-HLSLIntrinsicAttr 0x{{.+}} <<invalid sloc>> Implicit "op" "" 338
+// CHECK-NEXT:| |-FunctionTemplateDecl 0x{{.+}} <<invalid sloc>> <invalid sloc> IsValid
+// CHECK-NEXT:| | |-TemplateTypeParmDecl 0x{{.+}} <<invalid sloc>> <invalid sloc> class TResult
+// CHECK-NEXT:| | `-CXXMethodDecl 0x{{.+}} <<invalid sloc>> <invalid sloc> implicit IsValid 'TResult () const'
+// CHECK-NEXT:| |-FunctionTemplateDecl 0x{{.+}} <<invalid sloc>> <invalid sloc> ThreadIncrementOutputCount
+// CHECK-NEXT:| | |-TemplateTypeParmDecl 0x{{.+}} <<invalid sloc>> <invalid sloc> class TResult
+// CHECK-NEXT:| | |-TemplateTypeParmDecl 0x{{.+}} <<invalid sloc>> <invalid sloc> class Tcount
+// CHECK-NEXT:| | `-CXXMethodDecl 0x{{.+}} <<invalid sloc>> <invalid sloc> implicit ThreadIncrementOutputCount 'TResult (Tcount) const'
+// CHECK-NEXT:| |   `-ParmVarDecl 0x{{.+}} <<invalid sloc>> <invalid sloc> count 'Tcount'
+// CHECK-NEXT:| `-CXXDestructorDecl 0x{{.+}} <<invalid sloc>> <invalid sloc> implicit referenced ~EmptyNodeOutput 'void () noexcept' inline
+
+// CHECK:|-CXXRecordDecl 0x{{.+}} <<invalid sloc>> <invalid sloc> implicit referenced struct EmptyNodeOutputArray definition
+// CHECK-NEXT:| |-FinalAttr 0x{{.+}} <<invalid sloc>> Implicit final
+// CHECK-NEXT:| |-HLSLNodeObjectAttr 0x{{.+}} <<invalid sloc>> Implicit EmptyNodeOutputArray
+// CHECK-NEXT:| |-FieldDecl 0x{{.+}} <<invalid sloc>> <invalid sloc> implicit h 'int'
+// CHECK-NEXT:| `-CXXMethodDecl 0x[[SUB:[0-9a-f]+]] <<invalid sloc>> <invalid sloc> used operator[] 'EmptyNodeOutput (unsigned int)'
+// CHECK-NEXT:|   |-ParmVarDecl 0x{{.+}} <<invalid sloc>> <invalid sloc> index 'unsigned int'
+// CHECK-NEXT:|   |-HLSLIntrinsicAttr 0x{{.+}} <<invalid sloc>> Implicit "indexnodehandle" "" 12
+// CHECK-NEXT:|   `-HLSLCXXOverloadAttr 0x{{.+}} <<invalid sloc>> Implicit
+
+// CHECK: `-FunctionDecl 0x{{.+}} node_2_0 'void (EmptyNodeOutputArray)'
+// CHECK:   |-ParmVarDecl 0x[[Param:[0-9a-f]+]] {{.+}} used OutputArray_2_0 'EmptyNodeOutputArray'
```

Comment:
You stopped using `CHECK-NEXT` for the rest of these.  That's different than the way the use was checked in the other test.  Was that on purpose?